README.md

```diff
@@ -103,6 +103,395 @@ tns.cloudBuildService
+Definition:
+
+```TypeScript
+/**
+ * Invalidates the current user authentication data.
+ * @returns {void}
+ */
+logout(): void;
+```
+</br>
+Usage:
+
+```JavaScript
+const tns = require("nativescript");
+
+tns.authenticationService.logout();
+```
+
+* `isUserLoggedIn` - Checks if the access token of the current user is valid. If it is - the method will return true. If it isn't - the method will try to issue new access token. If the method can't issue new token it will return false.
+</br>
+
+Definition:
+
+```TypeScript
+/**
+ * CheChecks if there is user info and the access token of the current user is valid. The method will try to issue new access token if the current is not valid.
+ * @returns {Promise<boolean>} Returns true if the user is logged in.
+ */
+isUserLoggedIn(): Promise<boolean>;
+```
+</br>
+
+Usage:
+```JavaScript
+const tns = require("nativescript");
+
+tns.authenticationService
+	.isUserLoggedIn()
+	.then(isLoggedIn => console.log(isLoggedIn))
+	.catch(err => console.error(err));
+```
+
+* `refreshCurrentUserToken` - Uses the refresh token of the current user to issue new access token.
+</br>
+
+Definition:
+
+```TypeScript
+/**
+ * Uses the refresh token of the current user to issue new access token. 
+ */
+refreshCurrentUserToken(): Promise<void>;
+```
+</br>
+
+Usage:
+```JavaScript
+const tns = require("nativescript");
+
+tns.authenticationService.refreshCurrentUserToken()
+	.then(() => console.log("Success"))
+	.catch(err => console.error(err));
+```
+
+#### Interfaces:
+```TypeScript
+interface IUser {
+	email: string;
+	firstName: string;
+	lastName: string;
+}
+
+interface IAuthenticationService {
+	/**
+	 * Uses username and password for login and after successfull login saves the user information.
+	 * @param {string} username The username of the user.
+	 * @param {string} password The password of the user.
+	 * @returns {Promise<IUser>} Returns the user information after successful login.
+	 */
+	devLogin(username: string, password: string): Promise<IUser>;
+
+	/**
+	 * Opens login page and after successfull login saves the user information.
+	 * If options.openAction is provided, it will be used to open the login url instead of the default opener.
+	 * @param {ILoginOptions} options Optional settings for the login method.
+	 * @returns {Promise<IUser>} Returns the user information after successful login.
+	 */
+	login(options?: ILoginOptions): Promise<IUser>;
+
+
+	/**
+	 * Invalidates the current user authentication data.
+	 * @returns {void}
+	 */
+	logout(): void;
+
+	/**
+	 * Uses the refresh token of the current user to issue new access token. 
+	 */
+	refreshCurrentUserToken(): Promise<void>;
+
+	/**
+	 * Checks the token state of the current user.
+	 * @returns {Promise<ITokenState>} Returns the token state
+	 */
+	getCurrentUserTokenState(): Promise<ITokenState>;
+}
+
+interface ILoginOptions {
+	/**
+	 * Action which will be used to open the login url.
+	 */
+	openAction?: (loginUrl: string) => void;
```

Comment:
We don't need to await this promise because our logic does not depend on the opening of the login page. It depends on the redirect to the localhost server.